src/test/java/com/saucelabs/ci/sauceconnect/SauceConnectManagerTest.java

```diff
@@ -255,6 +255,25 @@ void testSauceConnectSecretsCoveredWithStars() {
 
   }
 
+  @Test
+  void testSauceConnectLegacySecretsCoveredWithStars() {
+    // FIXME: Remove when SC4 legacy support is removed
```

Comment:
Could you please create a task in jira for that?

Reply:
Already done :) SC-5138